js/index.js

```diff
@@ -0,0 +1,61 @@
+$(document).ready(function() {
+	//First we'll need our variables
+	var answer; //This will be the randomly generated number
+	//answer = Math.floor((Math.random() * 100) +1); 
+	//answer = 5;
+	var guess; //This will be the user's current guess
+	var count = 5; //This is the number of guesses remaining
+	//$('#reset').click(resetFunc);
+
+	$('button').click(function(event) {
+		event.preventDefault();
+            var button_type = ($(this).attr("value"));
+            //answer = Math.floor((Math.random() * 100) +1); 
+            //alert("Here is the number " + answer);
+            if (button_type === 'guess') {
+            	guess = parseInt($('#guess').val());
+            	main();
+            } else {
+            	resetFunc();
+            }
+        });
+
+	function resetFunc() {
+		answer = Math.floor((Math.random() * 100) +1); 
```

Comment:
@filippia This is good to re-generate a new answer after we've played the game, but where is it being generated when we first load the page?